src/components/Hero/Hero.module.scss

```diff
@@ -85,3 +85,49 @@
   height: 100%;
   background-color: rgba(0, 0, 0, 0.3);
 }
+
+@media (max-width: 600px) {
```

Comment:
when having media styles, it's better to colocate it with the desktop version. it makes refactoring easier/organized. So this media query should be nested within .heroFooter { above